acmw-db-app/pages/api/attendance/[pid].js

```diff
@@ -0,0 +1,65 @@
+import pgQuery from '../../../postgres/pg-query.js';
+//idk I wanna do this somehow but need to export it from student? thots?
+//import {createStudent} from '../student/[pid].js'; 
+
+// POST /api/attendance/record
+// Record a specific student attendance
+// event_code: code for meeting, f_name: first name of student, 
+// l_name_dot_num: last name and dot number of student, year_level: school level of student
+async function record (event_code, f_name, l_name_dot_num, year_level) {
+    const curr_student_id = await pgQuery(`SELECT id FROM student WHERE name_dot_num = '${l_name_dot_num}'`);
```

Comment:
since now in the request body the last name dot number could contain capital letters I think this needs to be '${l_name_dot_num.toLowerCase()}` (or whatever the js to lowercase method is)

Reply:
gotti